turbo.json

```diff
@@ -16,8 +16,10 @@
       "persistent": true
     },
     "storybook:build": {
-      "dependsOn": ["@webstudio-is/icons#build"],
-      "cache": false
```

Comment:
No reason to avoid caching this!

Reply:
Additionally, because of the bump to >1.7, you'll need to define your outputs.

Doc here: https://turbo.build/repo/docs/core-concepts/caching#configuring-cache-outputs